apps/src/JSInterpreter.js

```diff
@@ -73,7 +104,7 @@ var JSInterpreter = module.exports = function (options) {
   };
 
   try {
-    this.interpreter = new window.Interpreter(options.code, initFunc);
+    new window.Interpreter(options.code, initFunc);
```

Comment:
Don't we have a linting rule somewhere that complains about `new` calls with unused return values?  In any case, I'd appreciate a brief comment explaining what this does, and maybe why the `new` keyword is necessary.